src/connectors/create.js

```diff
@@ -0,0 +1,30 @@
+const vec3 = require('../math/vec3')
+
+/**
+ * FIXME remove this and add to user guide
```

Comment:
I think this would also be fine as a docstring to be honest: files > wikis, as they can be shipped anywhere, included in the code etc

Reply:
done